### PR TITLE
Add galaxyproject.org cache purge, update 404

### PR DESCRIPTION
**Pull request by dannon**
This will allow us to manually clear cache on hub builds, and it updates the 404 to point to the hub's 404 page.

### CODE REVIEW

File: env/galaxy/templates/nginx/galaxyproject.j2 [hunk 1]
```diff
@@ -4,6 +4,16 @@
 
 proxy_cache_path    /var/lib/nginx/cache levels=1:2 keys_zone=s3_cache:10m max_size=128m inactive=60m use_temp_path=off;
```
Comment (natefoo) [inline env/galaxy/templates/nginx/galaxyproject.j2:5]:
```suggestion
proxy_cache_path    /var/lib/nginx/cache levels=1:2 keys_zone=s3_cache:10m max_size=128m inactive=60m use_temp_path=off purger=on;
```

**Comment by natefoo** [comment]
One thing I think, the rest looks good?

File: env/galaxy/templates/nginx/galaxyproject.j2 (hunk 1 above)
Comment (dannon) [reply env/galaxy/templates/nginx/galaxyproject.j2:5]: Whoops!

**Comment by dannon** [comment]
For posterity -- this would have worked great, except it appears `purger` functionality is only available with the nginx commercial subscription.

**Comment by natefoo** [comment]
It was a good attempt.

**Pull request closed without merging**